source/flight_software/propulsion_system.rst

```diff
@@ -0,0 +1,63 @@
+
+The propulsion system consists of an inner tank (Tank1) and the (outer) thrust tank (Tank2). 
+There are two valves from Tank1 to Tank2 - the primary valve and the backup valve. We 
+refer to these valves as the Tank1 valves. 
+Tank2 has four valves that are arranged tetrahedrally. 
+The purpose of the propulsion system is to pressurize Tank1 and then open the valves in Tank2
+at a predetermined schedule. This will create a force in the opposite of direction. 
```

Comment:
Maybe say "at a schedule provided by GNC software" since it's not really predetermined ahead of time